src/GlobalStyles.js

```diff
@@ -1,17 +1,29 @@
 import { createGlobalStyle } from 'styled-components/macro';
-@import url('https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,600;1,100;1,200;1,300&display=swap');
 
 export default createGlobalStyle`
-  *{
+*, ::after, ::before {
```

Comment:
nice!

Reply:
That's better than what I usually do. Nice!